src/test/js/shared/idam_helper.js

```diff
@@ -504,8 +504,10 @@ class IdamHelper extends Helper {
     interceptRequestsAfterSignin() {
         const helper = this.helpers['Puppeteer'];
         helper.page.setRequestInterception(true);
+        const pages = ["/login", "/register", "/activate", "/verification"];
+
         helper.page.on('request', request => {
-            if (request.url().indexOf('/login') > 0 || request.url().indexOf('/register') > 0 || request.url().indexOf('/activate') > 0 || request.url().indexOf('/verification') > 0 | request.url().indexOf('/pin') > 0) {
+            if (pages.some(v => request.url().includes(v))) {
```

Comment:
lol I didn't realise it's javascript not java and recommended using stringutils 🤦‍♂️